packages/table-dev-app/src/mutableTable.tsx

```diff
@@ -211,6 +211,7 @@ export interface IMutableTableState {
     scrollToRegionType?: RegionCardinality;
     scrollToRowIndex?: number;
     selectedFocusStyle?: FocusStyle;
+    selectedRegions: IRegion[];
```

Comment:
this should be optional. i think that's the source of compiler errors.

but also you have lint failures.

Reply:
thanks